proposals/0383-allow-let-property-wrapper.md

```diff
@@ -0,0 +1,179 @@
+# Allow Property Wrappers on Let Declarations
+
+* Proposal: [SE-038N](NNNN-filename.md)
+* Authors: [Amritpan Kaur](https://github.com/amritpan), [Pavel Yaskevich](https://github.com/xedin)
+* Review Manager: TBD
+* Status: **Awaiting review**
+* Implementation: [apple/allow-let-property-wrapper](https://github.com/apple/swift/pull/62342) 
+
+## Introduction
+
+SE-0258 Property Wrappers models both mutable and immutable variants of a property wrapper but does not allow wrapped properties explicitly marked as a `let`. This proposal builds on the original vision by expanding property wrapper usage to include let declared properties with a wrapper attribute.
+
+## Motivation
+
+Allowing property wrappers to be applied to `let` properties improves Swift language consistency and code safety.
```

Comment:
"Safety" in Swift Evolution lingo usually refers to memory safety, and we've discouraged looser usages. Do you mean here perhaps _expressivity_?

Reply:
I replaced "safety" with "expressivity" here. Is there also a difference between "expressivity" and "expressiveness" for Swift Evolution lingo?